src/foam/u2/tag/Select.js

```diff
@@ -37,6 +40,7 @@ foam.CLASS({
       var self = this;
 
       this.attrSlot().linkFrom(this.data$);
+      this.attrs({ size: this.size$ });
```

Comment:
this.setAttribute('size', this.size$); would save an enumeration over the object, buts its probably pretty minor.